eng/targets/Cpp.Common.props

```diff
@@ -9,6 +9,12 @@
     <IntDir>$(IntermediateOutputPath)</IntDir>
   </PropertyGroup>
 
+  <!-- These test projects are skipped because you cannot test native ARM64 projects on a x86/x64 machine. -->
```

Comment:
@jkotalik You may want to specifically review this and the changes to the `Cpp.Common.Targets` as well